v2/session/serverless/with-aws-lambda/session-verification.mdx

```diff
@@ -33,16 +32,18 @@ import { getBackendConfig } from "./config.mjs";
 import { verifySession } from "supertokens-node/recipe/session/framework/awsLambda";
 import middy from "@middy/core";
 import cors from "@middy/http-cors";
+__OMIT_START__
```

Comment:
should add about this in the contributing.md file in the root of the docs repo

Reply:
Done